src/test/resources/features/users/create_users.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Create Users  
+
+  In order to create an account
+  As a valid Converter customer
+
+  @functional
+  Scenario: Verify that is possible to create a new account
+    When I send a POST request to "/user/createUser" with the following form data
+      | username   | laura     |
+      | name       | laura     |
+      | lastName   | toro      |
+      | password   | Secret123 |
+      | rePassword | Secret123 |
+    Then I validate the response has the "200" status code
+    And I validate that the response body should match with "users/createUsers.json" JSON schema
+    And I validate that the response contains the following values
+      | status  | 200                                                    |
+      | message | "Your account was created! Please, login to continue." |
+
+  @functional
+  @createUser
```

Comment:
Please put tags in a single line, first the corresponding ones for categorization, after the corresponding ones to preconditions or posconditions.
`@functional @createUser`

Reply:
Thanks, It's done.